### ISSUE TITLE
Unlisted dependencies @vitest/coverage-v8

### ISSUE THREAD

**Issue by gajus** [webpro/knip#347]
Error:

```
Unlisted dependencies (1)
@vitest/coverage-v8                vitest.config.ts
```

Subject file:

```ts
import path from 'node:path';
import { defineConfig } from 'vitest/config';

export default defineConfig({
  resolve: {
    alias: {
      '#': path.resolve(__dirname, './test'),
      $: path.resolve(__dirname, './testV2'),
      '@': path.resolve(__dirname, './src'),
    },
  },
  test: {
    coverage: {
      enabled: false,
    },
    environment: 'node',
    // Note that we are intentionally using *.test.ts suffix because
    // *.test.{ts,tsx} suffix is used by ESLint to identify vitest files.
    // Do not add non-*.test.{ts,tsx} files to the list below.
    include: ['./{src,test,testV2}/**/*.test.ts'],
    isolate: false,
    maxThreads: 5,
    minThreads: 1,
    mockReset: true,
    outputFile: './src/__generated__/vitest/results.xml',
    reporters: ['verbose'],
    setupFiles: path.resolve(__dirname, './test/vitest.setup.ts'),
    testTimeout: 60_000,
  },
});
```

I am assuming it sees `coverage` and thinks I am using it.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.41.3. See [Release 2.41.3](https://github.com/webpro/knip/releases/tag/2.41.3) for release notes._

**Comment by webpro** [comment]
Thanks! Indeed `coverage.enabled: false` was not respected.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v3.0.0-canary.4. See [Release 3.0.0-canary.4](https://github.com/webpro/knip/releases/tag/3.0.0-canary.4) for release notes._
